cmd/milmove/serve_test.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gofrs/uuid"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/transcom/mymove/pkg/auth"
+	"github.com/transcom/mymove/pkg/cli"
+	"github.com/transcom/mymove/pkg/logging"
+	"github.com/transcom/mymove/pkg/models"
+	"github.com/transcom/mymove/pkg/testingsuite"
+)
+
+type ServeSuite struct {
+	testingsuite.PopTestSuite
+	logger logger
+}
+
+func (suite *ServeSuite) SetupTest() {
+	suite.DB().TruncateAll()
+}
+
+func TestServeSuite(t *testing.T) {
+	flag := pflag.CommandLine
+
+	v := viper.New()
+	v.BindPFlags(flag)
+	v.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	v.AutomaticEnv()
+
+	logger, err := logging.Config(v.GetString(cli.DbEnvFlag), v.GetBool(cli.VerboseFlag))
+	if err != nil {
+		log.Panic(err)
+	}
+	hs := &ServeSuite{
+		PopTestSuite: testingsuite.NewPopTestSuite(testingsuite.CurrentPackage()),
+		logger:       logger,
+	}
+	suite.Run(t, hs)
+}
+
+func (suite *ServeSuite) TestIsLoggedInWhenNoUserLoggedIn() {
+	req := httptest.NewRequest("GET", "/is_logged_in", nil)
```

Comment:
It's nice when these match, though I'm pretty sure the path gets tossed anyway.

```suggestion
	req := httptest.NewRequest("GET", "/internal/users/is_logged_in", nil)
```